crates/shrs_core/src/builtin/help.rs

```diff
@@ -39,12 +39,19 @@ impl BuiltinCmd for HelpBuiltin {
         match &cli.command {
             Commands::Builtin => {
                 let cmds = sh.builtins.builtins.keys();
+                let info = sh.keybinding.get_info();
 
-                ctx.out.println("Builtin Commands")?;
+                ctx.out.println("Builtin Commands:")?;
 
                 for cmd in cmds {
                     ctx.out.println(cmd)?;
                 }
+
+                ctx.out.println("")?;
+                ctx.out.println("Key Bindings:")?;
```

Comment:
right now key bindings and builtins are shown together, it should be that you only display keybindings when you type `help keys` or `help bindings` or something like that. take a look at the `Commands` object on how to add a subcommand.

Reply:
I moved this functionality to now be invoked using `help bindings`. I attached a screenshot to this comment to demonstrate it.

![shrs2](https://github.com/MrPicklePinosaur/shrs/assets/47926823/7490ed06-5421-435f-9255-7cfa58ce7e1f)